### PR TITLE
doc: modernize README.md and add CoC

### PR DESCRIPTION
**Pull request by aqrln**
* Remove obsolete and misleading information from `README.md`.
* Make `README.md` look cleaner.
* Add a Code of Conduct.
* Add a simple introductory example to the front page and duplicate it
  in `examples/readme-example`.
* Add the list of maintainers.

image

### CODE REVIEW

**Comment by belochub** [comment]
@aqrln, please add `Refs: https://github.com/metarhia/JSTP/issues/163` to commit description.

**Comment by belochub** [comment]
@aqrln, this PR also addresses https://github.com/metarhia/JSTP/issues/157, this https://github.com/metarhia/JSTP/issues/157#issuecomment-300619954 in particular, but it is up to you whether to mention this in commit description or not.

**Comment by belochub** [comment]
@aqrln, additions of examples is connected to https://github.com/metarhia/JSTP/issues/131, you should also mention this in commit description.

**Comment by aqrln** [comment]
@belochub I've updated the commit message with appropriate references, thanks!

File: README.md [hunk 1]
```diff
@@ -1,91 +1,159 @@
...
 [jstp.min.js](https://metarhia.github.io/JSTP/dist/jstp.min.js) and
-[jstp.min.js.map](https://metarhia.github.io/JSTP/dist/jstp.min.js.map) are
-also available.
+[jstp.min.js.map](https://metarhia.github.io/JSTP/dist/jstp.min.js.map)
+available for those browser-based applications that aren't built using a module
+bundler like [webpack](https://webpack.js.org/).
+
+We also have official client-side implementations for
+[Swift](https://github.com/JSTPMobile/iOS) and
+[Java](https://github.com/JSTPMobile/Java) that work effortlessly on iOS and
```
Comment (lundibundi) [inline README.md]: Maybe it's better to put 'that work...' on a same line as 'Android', it may be easier to edit later + better diff in case we need to.

Comment (aqrln) [inline README.md]: No strong opinion about that, but done :)

**Comment by aqrln** [comment]
@metarhia/jstp-core given the facts that this PR has already been reviewed by all the possible maintainers (so there's no one to object or request changes anymore) and this is a change we'd like to happen ASAP ('cause the current README is kinda misleading), is it okay for me to go ahead and land this?

**Comment by lundibundi** [comment]
@aqrln I think we discussed that it can be landed right away it all maintainers positively reviewed the pr, ain't we?

**Comment by aqrln** [comment]
@lundibundi I don't recall us to have discussed that specific case, but it seems rather logical to me, so I'm gonna merge it right away. Please lmk if this is something that shouldn't have happened.

**Comment by aqrln** [comment]
Landed in f6a3e92c1eea2ba87f658e8618512c6efad048f1 🚀

**Pull request closed without merging**